### PR TITLE
Feature/sensor project dockerfile

### PR DESCRIPTION
**Pull request by larsfroelich**

### CODE REVIEW

**Comment by larsfroelich** [comment]
should be easy to merge since it's just a single file being added, but I'd like to fully test deployments using the dockerfile first before merging this. that's why it's a draft for now, it's not well-tested yet.
Will upgrade it to a normal PR soon and then we can merge it

**Review by patrickjaigner — APPROVED** [review]
Approved after Adam Runner confirmed, that we can use his gist snipped.

**Comment by larsfroelich** [comment]
> Approved after Adam Runner confirmed, that we can use his gist snipped.

I reached out to him and he agreed to his script being used in an open source project with no strings attached (except for attribution of course)

https://gist.github.com/adamrunner/285746ca0f22b0f2e10192427e0b703c?permalink_comment_id=4995387#gistcomment-4995387

**Pull request merged**
